### PR TITLE
[16.0][MIG] purchase_date_planned_manual: Migration to 16.0

### PR DESCRIPTION
**Pull request by chaule97**

### CODE REVIEW

**Review by HviorForgeFlow — APPROVED** [review]
Functional review

**Review by santostelmo — APPROVED** [review]

File: purchase_date_planned_manual/tests/test_date_planned_manual.py [hunk 1]
```diff
@@ -107,80 +114,90 @@ def test_manually_set_pol_date(self):
...
+        self._run_procurement(self.product_1, origin, values)
+        orders = self.env["purchase.order"].search([("origin", "=", origin)])
         self.assertEqual(
-            po_line_1.product_qty,
-            2.0,
-            "The first PO line product qty should still be 2.",
+            len(orders.order_line),
+            1,
+            "The PO should still have only one PO line.",
         )
```
Comment (LoisRForgeFlow) [inline purchase_date_planned_manual/tests/test_date_planned_manual.py:181]: ~ The second procurement in this test was key of the test and it has been removed... ~

EDIT: nevermind, I just saw that it was moved to a second test. My bad.

Comment (LoisRForgeFlow) [reply purchase_date_planned_manual/tests/test_date_planned_manual.py:181]: In any case, there was a bug. It is fixed here https://github.com/OCA/purchase-workflow/pull/2783, reviews are welcomed.

**Pull request merged**